packages/api-electron-demo/test/init.js

```diff
@@ -0,0 +1,51 @@
+var Application = require('spectron').Application;
+var should = require('chai').should();
+const {
+  before,
+  after,
+  describe,
+  it
+} = require('mocha');
+const path = require('path');
+const electronPath = path.join(__dirname, '..', 'node_modules', '.bin', 'electron.cmd');
+const liveServer = require('live-server');
+
+const params = {
+  port: 5000,
+  host: '127.0.0.1',
+  root: 'src',
+  open: false,
+  ignore: '*'
+};
+
+liveServer.start(params);
+
+describe('application launch', function() {
+  this.timeout(10000);
```

Comment:
We need to wait for electron/openfin to start before the tests start to run. The default is 2 seconds which is too short.